lib/utils.js

```diff
@@ -20,8 +20,79 @@ function getFnArgs(fn) {
     .filter((arg) => arg);
 }
 
+/*
+ * Map a brakes stats object to a hystrix stats object
+ */
+function mapToHystrixJson(json) {
+  const stats = json.stats;
+  return {
+    type: 'HystrixCommand',
+    name: json.name,
+    group: json.group,
+    currentTime: json.time,
+    isCircuitBreakerOpen: json.open,
+    errorPercentage: stats.failed / stats.total,
+    errorCount: stats.failed,
+    requestCount: stats.total,
+    rollingCountCollapsedRequests: 0, // not reported
+    rollingCountExceptionsThrown: 0, // not reported
+    rollingCountFailure: stats.failed,
+    rollingCountFallbackFailure: 0, // not reported
+    rollingCountFallbackRejection: 0, // not reported
+    rollingCountFallbackSuccess: 0, // not reported
+    rollingCountResponsesFromCache: 0, // not reported
+    rollingCountSemaphoreRejected: 0, // not reported
+    rollingCountShortCircuited: 0, // not reported
+    rollingCountSuccess: stats.successful,
+    rollingCountThreadPoolRejected: 0, // not reported
+    rollingCountTimeout: stats.timedout,
+    currentConcurrentExecutionCount: 0, // not reported
+    latencyExecute_mean: 13, // todo
```

Comment:
This can be `stats.latencyMean` I believe. Also have you testes to see what the dashboard looks like (if it loads) with the amount of stuff that isn't reported in this stream?